api/model/User.py

```diff
@@ -0,0 +1,64 @@
+from passlib.apps import custom_app_context as pwd_context
+
+
+class User:
+    def __init__(self, firstName, lastName, email, username, password, is_admin=False):
+        self.firstName = firstName
+        self.lastName = lastName
+        self.email = email
+        self.username = username
+        self.password = self.hash_password(password)
+        self.is_admin = is_admin
+
+    def getFirstName(self):
```

Comment:
@nanfuka Good work with the OOP. But unlike languages like Java, Its best to use snake case not camel cases in python when naming your methods, functions or variables for example here you could have named this like: get_first_name(self): and for the variable first_name